pkgs/by-name/xb/xbattbar/package.nix

```diff
@@ -4,19 +4,18 @@
   fetchgit,
   libX11,
   perl,
-  ...
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "xbattbar";
   version = "1.4.9";
 
   # The current active upstream of xbattbar seems to be the Debian source
   # repository.
   src = fetchgit {
     url = "https://salsa.debian.org/debian/xbattbar.git";
-    rev = "upstream/${version}";
-    sha256 = "10w7gs0l4hzhdn38yqyr3az7n4ncmfnd6hhhly6lk5dg7k441ck6";
+    rev = "upstream/${finalAttrs.version}";
```

Comment:
There is a matching tag for this: https://salsa.debian.org/debian/xbattbar/-/tags. 
This change should be squashed into the refactor commit.